docs/devel/flaky-tests.md

```diff
@@ -92,7 +92,14 @@ we have the following guidelines:
    back, at least 2 hours of merge time is wasted. So we need to make monotonic
    progress towards narrowing it down every time a flake occurs. If you can't
    figure it out from the logs, add log messages that would have help you figure
-   it out.
+   it out.  If you make changes to make a flake more reproducible, please link
+   your pull request to the flake you're working on.
+5. Once you have made a change that you believe fixes a flake, it is conservative
+   to keep the issue for the flake open and see if it manifests again after the
+   change is merged.
+6. If a flake has been open, could not be reproduced, and has not manifested in
+   2 months or so, it is reasonable to close the flake issue with a note saying
+   why.
```

Comment:
7. PRs that fix or help debug flakes may have the P0 priority set to get them through the merge queue as fast as possible.